v1 (#36)
cosmos/new-cosmos.network: content/articles/what-happens-to-atoms-when-they-are-staked.md

---
index: 9
title: What happens to ATOM when they are staked?

> description: When users stake ATOM, they effectively lock them up for an indefinite period of time.

Reviewer: ```suggestion
description: When ATOM are staked, the ATOM is effectively locked up for an indefinite period of time.
```

Reviewer: since the question is about the ATOM, we can answer about the ATOM (not the user)

  - blockchain
  - atom
---

> When users stake ATOM, they effectively lock them up for an indefinite period of time. This means ATOM cannot be transferred anymore, which is guaranteed by the protocol itself. However, users are free to trigger the process to unlock their ATOM at any point after they started staking (via the Wallet of their choice). This is called "undelegating". It will take 21 days for ATOM to be transferrable again after the transaction to undelegate has been sent.

Reviewer: ```suggestion
When you stake ATOM, you effectively lock the ATOM up for an indefinite period of time. This means ATOM cannot be transferred anymore, which is guaranteed by the protocol itself. However, you can trigger the process to unlock your ATOM at any point after you start staking using the wallet of your choice. This action is called "undelegating." ATOM is transferrable again 21 days after the transaction to undelegate has been sent.
```

Reviewer: can we use "you" here? I like it better... makes sense to always use "you"
